Mlem/Views/Shared/Links/Community Link View.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Community Link.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2023-06-27.
+//
+
+import Foundation
+import SwiftUI
+import CachedAsyncImage
+
+struct CommunityLinkView: View {
+    // TODO: setting
+    let showServerInstance: Bool = true
+    
+    let community: APICommunity
+    
+    var body: some View {
+        NavigationLink(value: community) {
+            HStack(alignment: .bottom, spacing: AppConstants.largeAvatarSpacing) {
```

Comment:
I think an HStack with Text and associated data is a strong indicator for wrapping them with `.accessibilityElement(children: combine)`.. probably on the HStack itsself.